usecases/FlexMex1_10/FlexMex1_10.py

```diff
@@ -0,0 +1,22 @@
+import os
```

Comment:
Maybe we can even abandon scripts for the usecases. The usecase specifics could be described in yaml files which are then processed in the functions and methods of the module. Only problem I see is that there are some irregularities for some use cases in the input data.